web/tests/schema.spec.ts

```diff
@@ -0,0 +1,81 @@
+import {test, expect} from "../playwright";
+
+const getTypeName = (header: string) => {
+  const secondPart = header.substring(header.indexOf("type") + 5);
+  const emptySpaceIndex = secondPart.indexOf(" ");
+  return secondPart.substring(0, emptySpaceIndex);
+};
+
+test.describe("schema", () => {
+  test.beforeEach(async ({page}) => {
+    await page.goto("_test/schema");
+  });
+
+  test("check basic UI", async ({uiClass}) => {
+    const filterControls = uiClass("textView_filterControls");
+    const filters = uiClass("textView_filterSelect");
+
+    // 2 filter dropdowns rendered
+    await expect(filters).toHaveCount(2);
+
+    // there is search input rendered
+    await expect(
+      filterControls.locator("//input[@placeholder='search...']")
```

Comment:
I believe we want to be able to run these tests with Playwright asap, so this is fine. But it is better to just use:
`filterControls.getByPlaceholder("search...") or
`filterControls.getByRole("textbox", { name: "search..." })
` if possible. Fixture `uiClass` I also good, but maybe it can be replaced at some places with `.getByRole` or another more semantic locator. In general, we should try to avoid locating by `css` and `XPath`s.